omniduct/databases/cursor_formatters.py

```diff
@@ -0,0 +1,146 @@
+import csv
+import io
+
+
+class CursorFormatter(object):
+
+    def __init__(self, db_client, cursor, **kwargs):
+        self.db_client = db_client
```

Comment:
I did originally, since there are some idiosyncrasies around the cursors provided by different libraries... however I ended up implementing the formatters in such a way as that those differences did not matter.

I guess I can remove the dependency here until I really need it. Fortunately, this can be removed (and/or readded) without changing any subclasses.

I'll dump it for now.